docs/troubleshooting.md

```diff
@@ -0,0 +1,43 @@
+# Troubleshooting
+
+This page collects a few known issues and solutions for common problems that users might encounter when using napari.
+If you are facing an error that is not covered by this page, please
+[open an issue](https://github.com/napari/napari/issues/new) in the napari repository.
```

Comment:
Maybe "check open issues to see if any are related, and if nothing is immediately obvious, open a new issue at..."?